mcs/class/corlib/System/TimeZoneInfo.cs

```diff
@@ -165,6 +165,12 @@ static TimeZoneInfo CreateLocal ()
 			}
 #endif
 
+#if UNITY
+			var localUnity = CreateLocalUnity ();
+			if (localUnity != null)
+				return localUnity;
```

Comment:
Do we really want this to kick in on all non-windows systems? we shouldn't return this here, and instead return it if  return FindSystemTimeZoneByFileName (tz, Path.Combine (TimeZoneDirectory, tz)); throws